File: 0217-contains-duplicate/0217-contains-duplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        HashSet<int> mySet = new HashSet<int>();
        for(int ptr = 0; ptr < nums.Length; ptr++){
            if(!mySet.Add(nums[ptr])){
                return true;
            }
            else{
                mySet.Add(nums[ptr]);
            }
        }
        return false;
    }
}

File: 0217-contains-duplicate/0217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Set<Integer> mySet = new HashSet<>();
        for(int ptr = 0; ptr < nums.length; ptr++){
            if(!mySet.add(nums[ptr])){
                return true;
            }
            else{
                mySet.add(nums[ptr]);
            }
        }
        return false;
    }
}